### ISSUE TITLE
Deprecated? 

### ISSUE THREAD

**Issue by philipp-heinrich** [doitintl/gceinstances#4]
Is this website no longer maintained? 
it throws an 404 error when visiting http://gceinstances.info

**Comment by Burekasim** [comment]
Yes, we replaced it with gcpinstances.info (different repo, website and domain)
